src/rollbar.php

```diff
@@ -122,6 +122,7 @@ class RollbarNotifier {
         'scrub_fields', 'shift_function', 'timeout', 'report_suppressed', 'use_error_reporting', 'proxy');
 
     // cached values for request/server/person data
+    private $_context = null;
```

Comment:
Sorta minor, but can you call this variable something besides `_context`? One of the fields that the Rollbar API understands is also "context" and I worry that this could be confused with that.

Just as some ideas, maybe `_execution_context` or `_php_context` ?